### ISSUE TITLE
What's the default value for pageRank? 

### ISSUE THREAD

**Issue by jackieros** [algolia/docsearch#1570]
Your docs make clear that higher ranked pages boost results and that it's possible to give a page any value including a negative number, but it's not clear what the default rank value is for pages that aren't given a hardcoded pageRank?  Where do those pages fall between higher & lower ranked pages?

Is the default value 0? 1? Something else? 

Also, what's the impact of giving a page a very high number - does it just put it first, or is there a weighted difference when giving a high number?  For example, if I rate all other pages in my site with values 1-4, and then rate one page a 5, would there be any difference in the results vs the pages ranked as '4' but with a better match to the search query, if instead of 5, I rated it 100?

**Comment by shortcuts** [comment]
Hey!

> it's not clear what the default rank value is for pages that aren't given a hardcoded pageRank?

There's no default, which means it's 0.

> Where do those pages fall between higher & lower ranked pages?

tldr: search results are sorted by weight (desc), so you can have both boosted and non boosted results.

The weight of each result will be computed for a given query (based on multiple factors: match level, position, etc.) and the `pageRank` value will be **added** to this final weight.

You can view the computed weight directly [from the dashboard](https://www.algolia.com/apps/YOUR_APP_ID/dashboard): "ranking criteria" icon bottom right of each record. It should give you an idea of what `pageRank` value is acceptable for you case.

> Also, what's the impact of giving a page a very high number - does it just put it first, or is there a weighted difference when giving a high number?
> For example, if I rate all other pages in my site with values 1-4, and then rate one page a 5, would there be any difference in the results vs the pages ranked as '4' but with a better match to the search query, if instead of 5, I rated it 100?

Yes, if you provide a huge `pageRank`, there's many chances that this record will always be first, no matter what the query is.

To come back on the weight answer above, DocSearch extracts records hierarchically (to try to match a page structure), so the weight is also computed on **which level** matches. Unless you want to promote a/many page(s), I would not recommend providing a huge value, as it might impact the search experience.

**Comment by jackieros** [comment]
Thanks Clement,

This explanation was super helpful! Suggest adding this information to the
documentation for this attribute :-)
__________________________________________________________
[image: Cloudinary] <https://cloudinary.com/>
*Jackie Rosenzveig  *   |   Director of Documentation  |  Cloudinary
***@***.***   |    cloudinary.com
[image: Watch Cloudinary in action]

**Comment by randombeeper** [comment]
@jackieros Well, a little later than you probably imagined, but I have added this great info from shortcuts into our docs: https://docsearch.algolia.com/docs/record-extractor/#boost-search-results-with-pagerank. Thank you for the suggestion!
